package.json

```diff
@@ -7,6 +7,7 @@
     "description": "specifies dependencies needed for developing taco",
     "version": "0.0.1",
     "devDependencies": {
+        "taco-simulate": "^0.1.1",
```

Comment:
Why have you added this here? We don't need taco-simulate in order to compile/develop the packages, we'd just need it as a dependency of taco-cli surely?